pkg/cli/config/config.go

```diff
@@ -96,6 +98,52 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	return bpConfig, nil
 }
 
+func getProxyURL(bpConfig BackplaneConfiguration, proxyURLConfig any) string {
+	// as part of OSD-19785, proxy-url in config file is converted to []string from string to allow multiple proxies.
```

Comment:
We do not need too descriptive comments, we can always traceback to the Jira lined to the commit if we need more details.

```suggestion
```

Reply:
Fixed.